docs/reference/ml/apis/find-file-structure.asciidoc

```diff
@@ -166,35 +166,35 @@ full `grok_pattern`.
 If this parameter is not specified, the structure finder chooses the best format from
 the formats it knows, which are these Joda formats and their Java time equivalents:
```

Comment:
For 6.7 I will flip this to say "these Java time formats and their Joda equivalents".  Then the formats in the list will change similarly to how you changed them.  But I would rather do this as part of the end-to-end update to the file structure finder rather than just replacing `Y` with `y` leaving the list in an inconsistent state.  For example, many of the `Z`s need changing to `X`s too.